src/components/DirectoryCityGrid.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import { twMerge } from "tailwind-merge";
+import { Address } from "@yext/pages/components";
+import { formatPhoneNumber, formatPhoneNumberIntl } from 'react-phone-number-input';
+
+
+export const directoryCityGridFields = [
+    "c_addressRegionDisplayName",
+    "dm_directoryParents.name",
+    "dm_directoryParents.slug",
+    "dm_directoryParents.meta",
+    "dm_directoryParents.c_addressRegionDisplayName",
+    "dm_directoryChildren.name",
+    "dm_directoryChildren.address",
+    "dm_directoryChildren.mainPhone",
+    "dm_directoryChildren.slug"
+];
+
+interface DirectoryGridProps {
+    name?: string;
+    description?: string;
+    directoryParents?: any;
+    directoryChildren?: any;
+    relativePrefixToRoot?: string;
+}
+
+
+const DirectoryCityGrid = ({ name, description, directoryChildren, relativePrefixToRoot } : DirectoryGridProps) => {
+    let sortedChildren;
+    let childrenDivs;
+    if (directoryChildren) {
+        // Order children alphabetically by postal code
+        sortedChildren = directoryChildren.sort(function(a:any, b:any) {
+        a = a.address.postalCode;
```

Comment:
As a human, I would assume and appreciate a list of cities to be sorted by the city name.